pontoon/sync/tasks.py

```diff
@@ -389,3 +400,13 @@ def sync_translations(
             locales=repo_locales[r.pk].exclude(code__in=failed_locales)
         )
     repo_sync_log.end()
+
+    if db_project.pretranslation_enabled:
```

Comment:
If a new file is added, both conditions below will be true. So I suggest we exclude `new_locales` from the 2nd `pretranslate()` call.

Reply:
In the 2nd call as well, we will only loop through the entity locale pairs without actually pretranslating them ( as well have pretranslated them above). Although we can pass the excluded locales to make it bit fast it doesn't create any difference in the pretranslations created. What do you think??

Reply:
I'm aware of that. I'd still avoid calling the method if it's not needed.